### PR TITLE
deps(sysdig,node-analyzer): bump vm/eveclient-api to v1.1.3

### PR DESCRIPTION
**Pull request by mech-pig**
## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [ ] Title of the PR starts with type and scope, (e.g. feat(agent,node-analyzer,sysdig-deploy):)
- [ ] Chart Version bumped for the respective charts
- [ ] Variables are documented in the README.md (or README.tpl in some charts)
- [ ] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [ ] All test files are added in the tests folder of their respective chart and have a "_test" suffix

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

File: charts/sysdig/Chart.yaml [hunk 1]
```diff
@@ -15,4 +15,4 @@ name: sysdig
 sources:
```
Comment (AlbertoBarba) [inline charts/sysdig/Chart.yaml]: U can revert the changes in the `sysdig` chart since it's deprecated and we should not maintain it anymore

Comment (mech-pig) [reply charts/sysdig/Chart.yaml]: reverted

**Review by AlbertoBarba — APPROVED** [review]
